nrf_modem/doc/full-dfu.rst

```diff
@@ -0,0 +1,113 @@
+Full Modem firmware update
+##########################
+
+The modem library supports updating the full modem firmware when initialized in Full DFU mode via the ``nrf_modem_full_dfu.h`` interface.
+When the library is initialized in Full DFU mode, the only operation that can be carried out is a full modem firmware update.
+In particular, sockets are not available to the application.
+
+It is possible to switch between Full DFU and Normal operation modes by re-initializing the library.
+
+Memory Requirements
+*******************
+
+The total size of all shared memory regions must be at least 8220 bytes.
+@note:
+When using NCS, library initialization is done via the glue, which will configure the library shared memory regions' size and position in RAM.
+Use the Kconfig options belonging to the glue to ensure that the total size of the library shared memory region is as large as necessary.
+
+Modem library initialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to update the full modem firmware, the library must be initialized in `FULL_DFU` mode as shown below::
+
+	/* Shutdown modem to prepare for DFU */
```

Comment:
Suggest using the code block:

.. code-block:: c
/* Shutdown modem to prepare for DFU */
nrf_modem_init(params, FULL_DFU_MODE);

Reply:
Fixed